coda/coda_mdstore/views.py

```diff
@@ -1,9 +1,8 @@
-import codecs
 import copy
 import uuid
 
-from urllib2 import urlopen
-from urllib import urlencode
+from urllib.request import urlopen
+from urllib.parse import urlencode
 try:
```

Comment:
We should just automatically start replacing these try/except blocks with `import json`.

Reply:
Will get rid of it.